src/main/java/ch/heigvd/dai/models/Pixel.java

```diff
@@ -35,4 +44,30 @@ public int getBlue() {
   public void setBlue(int blue) {
     this.blue = blue;
   }
+
+  /**
+   * Adds the color components of another pixel to the current pixel.
+   *
+   * @param other the pixel whose color components are to be added to the current pixel
+   */
+  public Pixel add(Pixel other) {
+    this.red += other.red;
```

Comment:
Is it safe if the color becomes greater than 255 ?

Reply:
Totally not! This is quite a dangerous method to use I agree, but quite handy for my use case